### DISCUSSION TITLE
Changelog tempates

### DISCUSSION THREAD

**Discussion by iorbita** [templaza/astroid-framework#507] [General]
Hi,
is there a changelog page for templates Zero and One?

Thanks,
Lorenzo

**Comment by Chacapamac** [comment]
https://github.com/templaza/astroid-framework/releases

**Comment by iorbita** [comment]
yes, I was aware about framework changelog, I thought there was a separate page changelog for templates, apparently not.

**Reply by Chacapamac** [reply]
I see, not sure if anything specific to the templates, that will be a plus. I mark that as an **Enhancement** for @sonvnn
